endpoints/openrtb2/amp_auction.go

```diff
@@ -86,7 +86,8 @@ func NewAmpEndpoint(
 		bidderMap,
 		nil,
 		nil,
-		ipValidator}).AmpAuction), nil
+		ipValidator,
+		empty_fetcher.EmptyFetcher{}}).AmpAuction), nil
```

Comment:
Why are we using an empty fetcher? We should be able to use stored responses on the AMP endpoint I would imagine.

Reply:
I'm not sure about this yet. Do we need to support stored response functionality in amp endpoint?